release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -1300,6 +1300,12 @@ The Amazon Web Services (AWS) Elastic File System (EFS) Technology Preview featu
 
 * Technology preview badges were displayed on the OpenShift Pipelines workflows in the web console, though OpenShift Pipelines was GA. The Technology preview badges are now removed. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1945153[*BZ#1945153*])
 
+* Previously, *Pipelines* you created in the git import flow for private repos failed to run. This happened because *Pipeline* ServiceAccount did not use secrets created by git import flow for private git repositories. With this update, you can add a secret name to the pipeline ServiceAccount's annotations, and add pipeline-specific annotations to the provided secret. As a result, *Pipelines* for private git repositories run successfully. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1970485[BZ#1970485])
```

Comment:
```suggestion
* Previously, any *Pipelines* you created in the git import flow for private repos failed to run. This happened because the *Pipeline* `ServiceAccount` object did not use secrets created by git import flow for private git repositories. With this update, you can add a secret name to the pipeline `ServiceAccount` object's annotations, and add pipeline-specific annotations to the provided secret. As a result, *Pipelines* for private git repositories run successfully. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1970485[BZ#1970485])
```

Reply:
Same comments given for L1573 in this PR: https://github.com/openshift/openshift-docs/pull/34411

Please incorporate them.

Reply:
Thanks for the suggestion @rolfedh. Incorporated it in the new commit.